### PR TITLE
[fix](multi-catalog)fix all avro,hudi and jdbc scanner class loader dependencies conflicts

### PR DESCRIPTION
**Pull request by wsjz**
## Proposed changes

add preload-extensions module, put all conflict dependencies to pom.xml in preload-extensions

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/be-java-extensions/java-udf/pom.xml [hunk 1]
```diff
@@ -47,6 +63,170 @@ under the License.
             <artifactId>reflectasm</artifactId>
             <version>1.11.9</version>
         </dependency>
+        <dependency>
```
Comment (AshinGau) [inline fe/be-java-extensions/java-udf/pom.xml:66]: Why add so many dependencies in java-udf?

**Comment by wsjz** [comment]
run buildall

**Comment by wsjz** [comment]
run external

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.95 seconds
 stream load tsv:          542 seconds loaded 74807831229 Bytes, about 131 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 insert into select:          29.1 seconds inserted 10000000 Rows, about 343K ops/s
 storage size: 17162015008 Bytes

**Comment by wsjz** [comment]
run external

**Comment by wsjz** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.21 seconds
 stream load tsv:          535 seconds loaded 74807831229 Bytes, about 133 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.3 seconds inserted 10000000 Rows, about 341K ops/s
 storage size: 17161912511 Bytes

**Review by CalvinKirs — APPROVED** [review]

**Comment by zy-kkk** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 44.83 seconds
 stream load tsv:          539 seconds loaded 74807831229 Bytes, about 132 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.5 seconds inserted 10000000 Rows, about 338K ops/s
 storage size: 17162330278 Bytes

**Comment by wsjz** [comment]
run p0

**Review by zy-kkk — APPROVED** [review]
LGTM

**Comment by hidataplus** [comment]
what diffrents about "fe/fe-core/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java" and "fe/be-java-extensions/preload-extensions/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java",???

**Pull request merged**